### PR TITLE
Update for paint.net

### PR DESCRIPTION
**Pull request by Link-Satonaka**
Didn't sit right with me that it "installed" the zip, then installed the binary, so I implemented the unzip function instead. I believe a few things in the old script were depreciated, that should be taken care of.

Wrote an uninstall script to go with it.

### CODE REVIEW

File: automatic/paint.net/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
+$url = '{{DownloadUrl}}'
+
+# Zipped installer
+Get-ChocolateyWebFile $packageName "$ENV:Temp\paint.net.install.zip" $url
+Get-ChocolateyUnzip "$ENV:Temp\paint.net.install.zip" "$ENV:Temp\pdn"
+$installFile = Get-ChildItem -Path "$ENV:Temp\pdn" -Recurse -Include "*.exe" | Select-Object -First 1
+
+# elevation bug
+#Install-ChocolateyPackage $packageName $fileType $silentArgs $installFile
+Start-Process -FilePath "$installFile" -ArgumentList "$silentArgs" -Verb Runas
```
Comment (ferventcoder) [inline automatic/paint.net/tools/chocolateyInstall.ps1]: Unfortunately there is so much more transparent to the package things that can happen with the calls through to install-chocolateyinstallpackage related to getting the user input variables and whether or not they want it to be silent. This is not going to work.

Comment (ferventcoder) [inline automatic/paint.net/tools/chocolateyInstall.ps1]: Not as it stands right now.

**Comment by Link-Satonaka** [comment]
I'll remove my `Start-Process` workaround and implement `Install-ChocolateyInstallPackage`

Then it will be acceptable?

**Comment by ferventcoder** [comment]
Perhaps. We'll see if it works. From what it feels like though, there may need to be some fixes in Choco, not necessarily this.

**Pull request closed without merging**